rails/README.md

```diff
@@ -0,0 +1,32 @@
+# Rails Style
+
+## Routing & Controllers
+
+#### Avoid `member` and `collection` routes
+
+```Ruby
+# Bad
+
+resources :catering_orders do
+  member do
+    post :cancel
+  end
+end
+
+
+# Good
+
+resources :catering_orders do
+  resource :cancellation, only: [:create]
+end
+
+# -or-
+
+resources :catering_orders
+resources :catering_order_cancellation, only: [:create]
+```
+
+##### Why?
+* Prevents existing controllers from becoming too busy
+* Eases the use of ember-data
```

Comment:
Is this because ember-data makes assumptions about being RESTful?

In some cases it's useful to have a separate domain model for a `catering-order-cancellation` because:

1. this guarantees that the `catering-order` doesn't need to be mutated accidentally,
2. you can create a cancellation without necessarily loading the `catering-order` in the store.

There are good patterns for action style routes in Ember, notably with [ember-api-actions](https://github.com/mike-north/ember-api-actions).